package-lock.json

```diff
@@ -5311,7 +5311,8 @@
           "version": "2.1.1",
           "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.1.1.tgz",
           "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
@n8rzz

Reply:
🤔 not sure why this would be changed here.  Please verify your node and npm versions match the ones listed in the `package.json` file.